sources/packages/backend/libs/integrations/src/models/common.model.ts

```diff
@@ -15,3 +15,11 @@ export class ProcessSummaryResult {
    */
   errors: string[] = [];
 }
+
+/**
+ * Single character Yes | No option.
+ */
+export enum YNOptions {
```

Comment:
I would prefer to have the complete name here as `YesNoOptions`, but it is just a suggestion.

Reply:
I thought the same but it is because we have the enum already existing as YesNoOptions and also if the integration in future needs YesNoOptions (Yes | No) the we may have to use alias in import so prefer to keep it as it is.